<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>If you are a developer - add your own game!</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>50</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Game name: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Rating range</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ratingSpinBox">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addGameButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white</string>
   </property>
   <property name="text">
    <string>Add Game</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Short description:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="shortDescriptionLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Developer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="developerLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Publisher:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="publisherLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Platforms:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="platformsLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>300</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Genres:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="genresLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Tags:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tagsLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Languages:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="languagesLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Categories:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="categoriesLineEdit">
      <property name="font">
       <font>
        <family>Gill Sans</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: cyan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
